.github/workflows/make_wheel_macOS_arm64.sh

```diff
@@ -11,7 +11,8 @@ export TF_NEED_CUDA=0
 
 python --version
 
-RUN python -m pip install --upgrade protobuf==3.20.0
+python -m pip install tensorflow-io
```

Comment:
Do we need to add the tfio to requirement.txt?

Reply:
> It's better to specify the version for TFIO

Each TF-io version has its own version of TensorFlow. TF-io is not compatible across versions. It would be better to have pip automatically install according to the current version of TensorFlow.

Reply:
> Do we need to add the tfio to requirement.txt?
This may not be appropriate because TFRA is a release that is as compatible as possible with all TensorFlow releases, whereas TF-io is not.